### DISCUSSION TITLE
60 seconds isn't 120 seconds

### DISCUSSION THREAD

**Discussion by 11-Fitzgerald** [xanderfrangos/twinkle-tray#495] [Ideas]
Someone noted previously that the time-out to dim minimum of 60 seconds is both too long  (suggest a 10 sec option or simply user-adjustable from a second to an hour or whatever)  and kind of meaningless  ....  it triggers at around two minutes or at a random time of its own choosing.  I need the same thing. This can't be a hard fix (can it?) but a new ver is out and the problem was ignored. So, TT uninstalled now and using alternative.

**Comment by xanderfrangos** [comment]
Twinkle Tray is a free, open source project that I maintain by myself in my free time. This is not a paid product that comes with support. All bug fixing, help, and improvements are motivated 3 things:
1. The written appreciation left by users for the thousands of hours I've put into this app.
2. The occasional, much appreciated donations I get.
3. The wonderful users who have provided ongoing localization for free.

I do not appreciate you being rude because I didn't address a small issue you have. I see every single issue and discussion opened on this repository. I don't ignore anything. I have a full-time job that has nothing to do with Twinkle Tray. I work on what I can when I can. If that's not enough for you, I lose nothing by you using another app. [Monitorian](https://apps.microsoft.com/store/detail/monitorian/9NW33J738BL0?hl=en-us&gl=us) is a great alternative with a more active developer.

**Reply by 11-Fitzgerald** [reply]
Many thanks. Sorry if I was rude, but I get that a lot so I must be. Will work on it. I appreciate v much you taking the time to explain. All the best.

**Comment by xanderfrangos** [comment]
@11-Fitzgerald  v1.15.0 has been released with fixes and improvements for the idle detection timing. It's much more adjustable and responsive and now.

**Comment by 11-Fitzgerald** [comment]
Thank you.  I've been using it for a few days.  It is an early Christmas present, just what I was hoping for. A couple of cups of coffee coming your way.

**Reply by 11-Fitzgerald** [reply]
Oh and tks for fixing   - 'resuming from idle dimming' intermittent issue-   in 15.1. Excellent.
